lib/std/crypto/chacha20.zig

```diff
@@ -421,7 +662,7 @@ test "crypto.chacha20 test vector 1" {
         0x6a, 0x43, 0xb8, 0xf4, 0x15, 0x18, 0xa1, 0x1c,
         0xc3, 0x87, 0xb6, 0x69, 0xb2, 0xee, 0x65, 0x86,
     };
-    const input = [_]u8{
+    const m = [_]u8{
```

Comment:
nitpick

that kind of minor/preference changes are not really recommended
better to keep the original blame and keep the changes to a minimum (better for the reviewers too)

Reply:
That goes with aligning the ChaCha code with the Salsa code. These functions are very similar, so minimizing the differences between them will eventually reduce the size and improve the clarity of future PRs.

(Salsa is going to have reduced-round versions, too, if only because this is needed for the scrypt function)